Sources/Styleguide/PointFreeBaseStyles.swift

```diff
@@ -422,18 +423,31 @@ private let _codeClass = CssSelector.class("code")
 private let codeStyles =
   _codeClass % (
     backgroundColor(.other("#fafafa"))
+      <> color(.other("#24292e"))
```

Comment:
This accounts for the verrry subtle screen shot changes. The older code color was a lil too dark?